acra-censor/handlers/handlers_util.go

```diff
@@ -358,6 +491,66 @@ func handleRangeCondition(patternNode, queryNode *sqlparser.RangeCond) bool {
 	return true
 }
 
+// matchOrderBy handles order by construction
+// return true if match otherwise false
+func matchOrderBy(patternNode, queryNode sqlparser.OrderBy) bool {
+
+	if len(patternNode) != len(queryNode) {
+		return false
+	}
+	for index, _ := range patternNode {
+		if !strings.EqualFold(patternNode[index].Direction, queryNode[index].Direction) {
+			return false
+		}
+		if !strings.EqualFold(sqlparser.String(patternNode[index].Expr), ColumnConfigPlaceholderReplacer) {
+			return false
```

Comment:
we doesn't need sqlparser.String, better if we will get value from node with explicit cast to correct type

Reply:
Thanks. Agree